examples/next-app/components/Voice.tsx

```diff
@@ -13,6 +18,51 @@ export const Voice = ({ accessToken }: { accessToken: string }) => {
         // eslint-disable-next-line no-console
         console.log('message', message);
       }}
+      onToolCall={async (
+        toolCall: ToolCall,
+      ): Promise<ToolError | ToolResponse> => {
+        // eslint-disable-next-line no-console
+        console.log('toolCall', toolCall);
+
+        if (toolCall.name === 'weather_tool') {
+          try {
+            const result = await fetch(
+              'https://httpbin.org/status/400',
+              // 'https://api.weather.gov/gridpoints/TOP/31,80/forecast',
```

Comment:
can we reverse this so that the weather endpoint is the one that isn't commented out?